src/main/java/mateAcademy/device/Device.java

```diff
@@ -0,0 +1,74 @@
+package mateAcademy.device;
+
+import mateAcademy.accessory.Accessory;
+import mateAcademy.test.Test;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity(name = "device")
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+public abstract class Device {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id_device")
+    private Long id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "cost")
+    private Double cost;
+    @OneToMany(mappedBy = "device", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    private Set<Accessory> accessories;
+    @OneToMany(mappedBy = "device", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    private Set<Test> tests;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
this setter must have Long id - not primitive as you have used, and change it everywere where you have used primitive instead of wrapper type